spec/models/user_spec.rb

```diff
@@ -1,5 +1,25 @@
+# frozen_string_literal: true
+
 require 'rails_helper'
 
-RSpec.describe User, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+feature 'check user db table' do
```

Comment:
This should be a model spec, not a feature

Reply:
I understand